### ISSUE TITLE
Appalling lack of support.

### ISSUE THREAD

**Issue by UKenGB** [xteve-project/xTeVe#363]
I had some questions about how to get xTeVe up and running so asked on the Discord support channel. Apart from one user who finally helped, the official xTeVe 'support' consisted of saying it was a Plex problem, simply because I mentioned I would be using xTeVe with Plex. Requests for info on what xTeVe required in the m3u were met with misguided platitudes about just being a 'standard' m3u, when there is NO SUCH THING.

I realise the app is free, but they do say they offer support on Discord. But that is a lie and their attitude has left me incensed.

Basically useless imbeciles and if I possibly could, I would avoid xTeVe like the plague. 

Good luck getting any support.

**Comment by MCTyler** [comment]
![image](https://user-images.githubusercontent.com/6390898/232684072-3686b003-5224-4492-ac36-d83ef85c7008.png)

**Comment by alturismo** [comment]
> I had some questions about how to get xTeVe up and running so asked on the Discord support channel. Apart from one user who finally helped, the official xTeVe 'support' consisted of saying it was a Plex problem, simply because I mentioned I would be using xTeVe with Plex. Requests for info on what xTeVe required in the m3u were met with misguided platitudes about just being a 'standard' m3u, when there is NO SUCH THING.

as i ve been the user (im also from the "official" xteve support team who spares his free time to help others ...) who showed you the sample m3u entry so you have an sample.

but to make it clear, as i told you also in your support question, a basic standard m3u entry is all you need, what you really asked about are the extended m3u possibilties (which i also told you) and which are not needed (what i also commented).

and yes, m3u is a standard, there are extended m3u possibilities ... but as mentioned and also told to you, all you needed is a standard simple m3u.



> I realise the app is free, but they do say they offer support on Discord. But that is a lie and their attitude has left me incensed.
> 
> Basically useless imbeciles and if I possibly could, I would avoid xTeVe like the plague.
> 
> Good luck getting any support.

and sorry to say, but we do this since some time now and comments like this ... well, doesnt really motivate ;)

and btw, as you dont use the xepg function anyhow (you wanted to use plex epg), there are plenty Alternatives out there to load a standard m3u ;) and use it as "proxy" for plex, so no, you are not bound to xteve at all ;)

**Comment by UKenGB** [comment]
> ![image](https://user-images.githubusercontent.com/6390898/232684072-3686b003-5224-4492-ac36-d83ef85c7008.png)

I was hoping to discuss it with you MCTyler as you seem to know what you're talking about with regard to xTeVe.

However, I have to take you to task about the dismissive response I received on Discord. I asked 3 questions about xTeVe having said I would be using it with Plex (ie, for what it was originally developed). For that to generate the response of 'talk to Plex support' is simply idiotic. xTeVe was developed specifically to deal with Plex who actually have NOTHING to do with xTeVe or how it operates. On what planet does that mean questions about xTeVe should be aimed at Plex support.

**Comment by UKenGB** [comment]
> > I had some questions about how to get xTeVe up and running so asked on the Discord support channel. Apart from one user who finally helped, the official xTeVe 'support' consisted of saying it was a Plex problem, simply because I mentioned I would be using xTeVe with Plex. Requests for info on what xTeVe required in the m3u were met with misguided platitudes about just being a 'standard' m3u, when there is NO SUCH THING.
> 
> as i ve been the user (im also from the "official" xteve support team who spares his free time to help others ...) who showed you the sample m3u entry so you have an sample.
> 
> but to make it clear, as i told you also in your support question, a basic standard m3u entry is all you need, what you really asked about are the extended m3u possibilties (which i also told you) and which are not needed (what i also commented).
> 
> and yes, m3u is a standard, there are extended m3u possibilities ... but as mentioned and also told to you, all you needed is a standard simple m3u.
> 
> > I realise the app is free, but they do say they offer support on Discord. But that is a lie and their attitude has left me incensed.
> > Basically useless imbeciles and if I possibly could, I would avoid xTeVe like the plague.
> > Good luck getting any support.
> 
> and sorry to say, but we do this since some time now and comments like this ... well, doesnt really motivate ;)
> 
> and btw, as you dont use the xepg function anyhow (you wanted to use plex epg), there are plenty Alternatives out there to load a standard m3u ;) and use it as "proxy" for plex, so no, you are not bound to xteve at all ;)

Well, here we go.

There is NO standard m3u. Sorry, but there is NO standard that has ever been defined. A de facto standard has developed, but that's all it is and indeed, that only encompasses 2 'fields' on the EXTINF line. Time and Title. If you want to call anything a standard, that is it and it is not sufficient for xTeVe as it holds no info on channel name, number etc.

Those are incorporated into the name:value pairs following the 'time' field, but there is absolutely no standard for those. That would be a bit like claiming some form of standard for an XML file's contents.

In prior reading of xTeVe documentation I came across 2 entirely different ways to include the channel no/id. So with ambiguous documentation like that and inaccurate and nonsensical statements about 'any standard m3u will do', I had to ask just what data xTeVe wanted to see in that line. I can easily write anything I choose as it's all handled automatically from my database of channels, but I want to use suitable names/labels to ensure xTeVe could read them. For anyone who actually knows the answer, this would entail a 2 second response. The fact that I was so abruptly dismissed with claims I should ask Plex indicates to me that no-one answering on Discord actually knows. I wouldn't mind if the response was simply "sorry, I don't know", but to try and palm me off by telling me to ask Plex (who have NOTHING to do with this) was a dereliction of responsibility. If you cannot support the product, don't pretend you can (not you altruismo).

I have some interesting and worthwhile ideas of how to utilise xTeVe and hoped to discuss these with others with more experience and clarify some issues in my mind about how xTeVe operates. Sadly that was not to be and has left a very sour taste in my mouth from the whole experience. Shame as I was really pleased to be working with it and the original developer seemed genuinely helpful, but I guess he's moved on. More's the pity.

I'll figure it out, with xTeVe or an alternative but without any help from xTeVe's NO SUPPORT bods.

**Comment by alturismo** [comment]
> There is NO standard m3u. Sorry, but there is NO standard that has ever been defined. A de facto standard has developed, but that's all it is and indeed, that only encompasses 2 'fields' on the EXTINF line. Time and Title. If you want to call anything a standard, that is it and it is not sufficient for xTeVe as it holds no info on channel name, number etc.

all you need is the basic standard entry for an m3u, nothing more .. anything else is "nice" and "extended"

nevermind, lets end this discussion, i hope you get what you wanted and feel free to ask, may next time ask properly what you want and you rather will get an decent answer.

cause as more i read now, you are asking what would be the best m3u experience ... what is different to

"What special Format does xteve expect" to work .... when you think about it, these are 2 different storys.

And again, xteve doesnt expect anything special ... only nice to have is what i told you already, in terms you use an external epg look forward for matching id's, otherwise you have to map manually ...

![image](https://user-images.githubusercontent.com/8406490/232864498-d0b35155-4487-44b8-904a-701457e6a466.png)

**Comment by UKenGB** [comment]
I agree, there's no point in prolonging this, but I will just say that the Wikipedia page you linked to also starts by explaining that m3u is NOT a standard.

However, to explain my question, that excerpt above shows the example of "logo=" which seems clear, but also "tvg-logo=" is often shown as the label to use. What I was trying to ascertain is what actual label(s) does xTeVe understand. It's not hard to understand, but if it only uses "logo=" then I would be wasting my time writing m3us that use "tvg-logo=".

That's all I needed to know. What does xTeVe need/require/understand in the m3u file. But no-one was willing to even discuss it, trying to fob me off with "talk to Plex". In fact, it occurs to me that xTeVe is quite possibly written to look for any label that includes 'logo' to point to the logo file to use. Again, that's all I needed to know, without anyone being rudely dismissive.

That is just an example of what I was after, as channel-no, channel-name etc have the same ambiguity. What label(s) can xTeVe understand. Possibly many, but if I used the label "gobbledygook=" followed by the logo file path, I'm pretty sure xTeVe would not get it. So it is important to know what labels xTeVe can and will understand.

Not a Plex problem, a specifically xTeVe question and certainly not OFFTOPIC. I will try to find the time to peruse xTeVe code to see what I can glean from there.

Anyway, thanks @alturismo , best we leave it there eh?

**Comment by MCTyler** [comment]
```Anyway, thanks @alturismo , best we leave it there eh?```
Please close this issue...Thanks!

**Comment by ayancey** [comment]
Wow, this support is really bad! I hope you get your money back!

**Comment by alturismo** [comment]
> Wow, this support is really bad! I hope you get your money back!

its free, its open source, its just a bad joke to comment like this ... ;)

**Comment by pkhuynh24** [comment]
Tried using the link for the discord invite but it said it is expired. What is the current link?

**Comment by alturismo** [comment]
https://discord.com/invite/8uHrqZG5

**Comment by pkhuynh24** [comment]
> https://discord.com/invite/8uHrqZG5

Hi alturismo, thank you for the quick response.  Still get the link is invalid.
![IMG_6870](https://github.com/user-attachments/assets/f968cd54-b3d3-418a-bcfd-f25dcad15315)

**Comment by pkhuynh24** [comment]
Hi alturismo, I realized what is going on.  I am banned from the guild is the wording on my desktop.  Not sure how that happened.   Is there anyway I can be unbanned and ask for help.  

User - gotujockin(1533)

**Comment by dnsforge-repo** [comment]
@pkhuynh24 Discord ban has been appealed and you should be able to join the discord server.

**Comment by pkhuynh24** [comment]
Thank you! I am in!! Looking forward 😀
